toolkit/tools/imageconfigvalidator/imageconfigvalidator_test.go

```diff
@@ -35,11 +35,20 @@ func TestShouldSucceedValidatingDefaultConfigs(t *testing.T) {
 
 			fmt.Println("Validating ", configPath)
 
-			config, err := configuration.Load(configPath)
+			config, err := configuration.LoadWithAbsolutePaths(configPath, configDirectory)
 			assert.NoError(t, err)
 
+			if err != nil {
```

Comment:
>if err != nil { [](start = 3, length = 15)

Doesn't `assert.NoError(t, err)` halt the execution?

Reply:
No, the normal assert checks just accumulate error reports that print when the test completes. You can use `assert.FailNow()` and friends to halt.

---
In reply to: [549908109](https://github.com/microsoft/CBL-Mariner/pull/480#discussion_r549908109) [](ancestors = 549908109)